setup.py

```diff
@@ -22,18 +23,18 @@
     BASE_ENV = '/'
 
 NAPPS_PATHS = {
-    'main': os.path.join(BASE_ENV, 'var/lib/kytos/napps')
+    'main': join(BASE_ENV, 'var/lib/kytos/napps')
 }
-NAPPS_PATHS['installed'] = os.path.join(NAPPS_PATHS.get('main'), '.installed')
-NAPPS_PATHS['enabled'] = os.path.join(NAPPS_PATHS.get('main'), 'kytos')
+NAPPS_PATHS['installed'] = join(NAPPS_PATHS.get('main'), '.installed')
+NAPPS_PATHS['enabled'] = join(NAPPS_PATHS.get('main'), 'kytos')
```

Comment:
But the os.path is not a linter error, right ? So join is to much generic.. os.path.join makes more sense than only join.